src/UI/templates/default/Button/button.less

```diff
@@ -1,5 +1,10 @@
 .btn {
   font-size: @il-button-font-size;
+  &:focus,
```

Comment:
Wenn Du das jetzt noch in die Mainbar ziehst, finde ich das sehr, sehr gut!

Reply:
jep,is drin! neuen PR?

Reply:
bzw. raus aus der button.less und jetzt begrenzt auf meta- und mainbar